serverless/src/errorLogger/handler.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Logs an error reported by a client
+ * @param {Object} event
+ */
+const errorLogger = async (event) => {
+  const { body } = event
+  const { params = {} } = JSON.parse(body)
+
+  console.error('Error reported', params)
```

Comment:
I think we should break this up into a message with a known format so that we can setup alerts in Splunk. I'd suggest something like `[uid] (code) message` or something.

Reply:
I'm assuming that the message here is the one that included the action, so that will be an additional thing we can setup in our alerts/reports which would be awesome.